mm-bot/src/services/overall_fee_calculator.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+
+from web3 import Web3
+
+from models.order import Order
+from services.ethereum import create_transfer, estimate_transaction_fee, get_gas_price
+from services.withdrawer.ethereum_withdrawer import EthereumWithdrawer
+
+
+async def estimate_overall_fee(order: Order) -> int:
```

Comment:
```suggestion
# Operational cost per order done by the market maker. This includes calling the transfer 
# (from YABTransfer) + withdraw (from YABTransfer) + msg fee paid to Starknet (when calling withdraw)
async def estimate_overall_fee(order: Order) -> int:
```

Reply:
Now the method includes this description
```
    Operational cost per order done by the market maker.
    This includes:
        calling the transfer (from YABTransfer) +
        withdraw (from YABTransfer) +
        msg fee paid to Starknet (when calling withdraw)
 ```